package.json

```diff
@@ -11,7 +11,7 @@
     "e2e": "ng e2e --aot",
     "deploy": "ng build --prod && firebase deploy",
     "coverage": "ng test --watch=false --code-coverage",
-    "package": "npm run package:ngx-carbon-ad && npm run package:ngx-code-example && npm run package:ngx-debounce-click && npm run package:ngx-eq && npm run package:ngx-in-viewport && npm run package:ngx-input-datepicker && npm run package:ngx-input-range && npm run package:ngx-input-star-rating && npm run package:ngx-input-switch && npm run package:ngx-input-tag && npm run package:ngx-json-ld && npm run package:ngx-loaders && npm run package:ngx-nav-drawer && npm run package:ngx-progress && npm run package:ngx-tabs",
+    "package": "./node_modules/.bin/run-s package:*",
```

Comment:
I think you can just use `run-s package:*`. The node modules bin folder is added to `PATH` automatically for npm scripts.